### ISSUE TITLE
Drop tab/space UI from preferences

### ISSUE THREAD

**Issue by cassidyjames** [elementary/code#462]
Since we have it in the format bar in the header bar, I think we can drop it in the global preferences.

**Comment by danrabbit** [comment]
I think there was supposed to be a difference like the header one is for the current document and the one in prefs is the global default

**Comment by cassidyjames** [comment]
Ah. Right now they are bound together so there's not a difference. So I'm :man_shrugging:. We could drop the global one (then bring it back if we ever differentiate the two), or we could leave it as-is. It's just a little confusing with the same setting in two places right now.

**Comment by danirabbit** [comment]
Yeah personally I think we should have no plugins and just have features or not have them. The plugin set up makes really confusing cases like this where it's hard to know if a preference is going to have an effect or not

**Comment by jeremypw** [comment]
My inclination would be to manage tabs entirely from the format bar, incorporating editor-config management and preferences there.
